stylesheet.css

```diff
@@ -1,22 +1,83 @@
 *{
-    margin: 0;
-    padding: 0;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html, body{
+height: 100%;
+}
+
+body{
+display: flex;
+align-items: center;
+justify-content: center;
+flex-direction: column;
+background-color: aliceblue;
+}
+
+.navBar, #grid-container{
+background-color: rgba(100, 82, 82, 0.897);
+width: 800px;
+border-radius: 3px;
+}
+
```

Comment:
- Since writing CSS code with good style guide in mind is a habit that sales a developer, how about you introduced some style guide to help, came across this article [5 things to consider](https://blog.logrocket.com/5-things-to-consider-when-creating-your-css-style-guide-7b85fa70039d/#:~:text=A%20CSS%20style%20guide%20is,more%20consistent%20and%20maintainable%20project.) how about looking it up and see if you can change one or two, what you think? 😉

Reply:
Ok sir thank you